### PR TITLE
Improve variable & function names

### PR DESCRIPTION
**Pull request by katrinleinweber**
Just a bunch of renamings that hopefully make the purpose of each renamed element clearer.

- [x] also update https://github.com/swcarpentry/r-novice-inflammation/blob/master/_episodes_rmd/14-supp-call-stack.Rmd#L130
- [x] search other episodes

### CODE REVIEW

File: _episodes_rmd/02-func-R.Rmd [hunk 1]
```diff
@@ -285,11 +285,11 @@ We have one more task first, though: we should write some [documentation]({{ pag
 A common way to put documentation in software is to add [comments]({{ page.root }}/reference.html#comment) like this:
 
 ```{r}
-center <- function(data, desired) {
+center <- function(data, around) {
   # return a new vector containing the original data centered around the
-  # desired value.
+  # around value.
```
Comment (diyadas) [inline _episodes_rmd/02-func-R.Rmd]: I agree that this change makes sense and I like this PR a lot. The only point of confusion is that this now reads "original data centered around the around..." which might take a moment.

Perhaps: original data centered around the value of "around" ? A few more words.

Comment (katrinleinweber) [reply _episodes_rmd/02-func-R.Rmd]: Yeah 😅 Or `mid_point` as the parameter name?

Comment (diyadas) [reply _episodes_rmd/02-func-R.Rmd]: I like that a lot!

Comment (katrinleinweber) [reply _episodes_rmd/02-func-R.Rmd]: Done :-) Thanks for confirming!

**Comment by katrinleinweber** [comment]
I found another occurence of `fence` thanks to https://github.com/swcarpentry/r-novice-inflammation/issues/424#issuecomment-489684648 and renamed `outside` to `edges` because that seems more meaningful for the 1st & last element of a vector ;-)

**Comment by katrinleinweber** [comment]
I would like to have this included in #415, so I'll merge now. Feel free to revert if you disagree.

**Pull request merged**